### PR TITLE
chore: [IOPID-2077] Add correct title in settings section

### PR DESCRIPTION
**Pull request by Ladirico**
## Short description
In order to have a correct display of the title in the header between the display in ‘profile’ (first-level header) and that in ‘settings’ (second-level header), it was necessary to change the rendering logic of the entire section.  

## List of changes proposed in this pull request
- Create component that contains all logic and core render of the screen
- Create component that renders the correct wrapper component of the screen

## Demo
<p>

| 🤖 Android 🤖 | 🍏 iOS 🍏 |
| - | - |
| <video src="https://github.com/user-attachments/assets/2eb2617c-093a-4a53-9ff8-96c8ea5e8eee"/> | <video src="https://github.com/user-attachments/assets/e9787ef7-4b2d-48d0-b3fc-10a4b739c86a"/> |

</p>

## How to test
Run the application and follow the demo instructions

### CODE REVIEW

**Review by shadowsheep1 — APPROVED** [review]
LGTM

File: ts/screens/profile/ProfileMainScreen.tsx [hunk 1]
```diff
@@ -294,6 +264,48 @@ const ProfileMainScreen = () => {
       {(isDebugModeEnabled || isDevEnv) && <DeveloperModeSection />}
       {/* End Page */}
       <VSpacer size={24} />
```
[the remark this answers is not in the document]
Comment (Ladirico) [reply ts/screens/profile/ProfileMainScreen.tsx:266]: Ok perfect! Thanks!

File: ts/screens/profile/ProfileMainScreen.tsx [hunk 2]
```diff
@@ -294,6 +264,48 @@ const ProfileMainScreen = () => {
...
+  );
+
+  if (isSettingsVisibleAndHideProfile) {
+    return (
+      <IOScrollViewWithLargeHeader
+        title={{
+          label: isSettingsVisibleAndHideProfile
+            ? I18n.t("global.buttons.settings")
+            : ""
+        }}
```
[the remark this answers is not in the document]
Comment (Ladirico) [reply ts/screens/profile/ProfileMainScreen.tsx:292]: Yes, good catch!

File: ts/screens/profile/ProfileMainScreen.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ladirico) [reply ts/screens/profile/ProfileMainScreen.tsx]: done!

File: ts/screens/profile/ProfileMainScreen.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ladirico) [reply ts/screens/profile/ProfileMainScreen.tsx]: done!

**Review by ChrisMattew — APPROVED** [review]

**Pull request merged**